spring-cloud-alibaba-examples/spring-cloud-ai-example/README-en.md

```diff
@@ -33,7 +33,7 @@ The Spring Cloud Alibaba AI module is based on [Spring AI 0.8.1](https://docs.sp
    spring:
      cloud:
        ai:
-         tongyi:
+         tong-yi:
```

Comment:
why was this changed and have you tested this locally?

Reply:
Typo prompted by ide: In word 'tongyi' It seems that it is not a big problem, I withdraw it